connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -247,6 +249,27 @@ public boolean isSinkConnector(String connName) {
         }
     }
 
+    /*
+     * Retrieves the connector type by connector name
+     *
+     * @return the ConnectorType for the given connector name.
+     * @throws ConnectException if the ConnectorType cannot be determined
+     */
+    public ConnectorType connectorType(String clsName) {
```

Comment:
I would suggest moving this logic into `Plugins` so that we can factor out of `Plugins.newConnector` a common method to load the `Connector` class. That would ensure that we are also handling the alias logic. 

cc @kkonstantine